contracts/stellar-interchain-token-service/src/contract.rs

```diff
@@ -311,6 +311,17 @@ impl InterchainTokenServiceInterface for InterchainTokenService {
 
         Ok(())
     }
+
+    #[only_owner]
+    #[allow_during_migration]
```

Comment:
```suggestion
```
this is a normal entrypoint, it's not being called during upgrade but after it for each token